server/service/osquery_utils/queries.go

```diff
@@ -338,14 +338,22 @@ func discoveryTable(tableName string) string {
 	return fmt.Sprintf("SELECT 1 FROM osquery_registry WHERE active = true AND registry = 'table' AND name = '%s';", tableName)
 }
 
+const usersQueryStr = `WITH cached_groups AS (select * from groups) 
```

Comment:
Interesting ... I thought MySQL pre version 8 didn't support CTEs - TIL

Reply:
This is osquery (so, sqlite).

Reply:
Ah, gotcha - thanks